script.py

```diff
@@ -238,17 +238,11 @@ def returnRepository(owner, repository, limit):
 
     rows_to_fecth = ""
 
-    for column in columns:
-        if column == columns[-1]:
-            rows_to_fecth += f"repository.{column}"
-        else:
-            rows_to_fecth += f"repository.{column}, "
-
     sql = f"""
     SELECT
         {rows_to_fecth}
```

Comment:
Como tu tirou o código em cima, vai ter que tirar essa variavel pq ela ta vazia. Tem que colocar pra ele recuperar as colunas owner e repository

Reply:
Não entendi.

Reply:
Essa variavel rows_to_fetch é setada nessa for que tu tirou. Ela é setada como "". Pode substituir ela por owner, repository. Só precisa ser retornado esses dois valores